### PR TITLE
Allow multiple JDKs in smoke test on main

### PR DESCRIPTION
**Pull request by stefanvodita**
Like #13108, but targeting the `main` branch.

### CODE REVIEW

**Review by uschindler — APPROVED** [review]

**Comment by dweiss** [comment]
Please merge this, I'll just follow-up on top of that.

**Review by rmuir — APPROVED** [review]

**Comment by uschindler** [comment]
I will just push the change to main.

**Comment by uschindler** [comment]
Sorry for last 2 commits. In 9.x the CONTRIBUTING.md was missing. All fine. Sorry for this.

I compared both files, all is fine and minor changes are expected.

**Comment by uschindler** [comment]
Thanks for merging this :-)

File: dev-tools/scripts/smokeTestRelease.py [hunk 1]
```diff
@@ -39,6 +39,8 @@
 from collections import namedtuple
 import scriptutil
 
+BASE_JAVA_VERSION = "17"
```
[the remark this answers is not in the document]
Comment (uschindler) [reply dev-tools/scripts/smokeTestRelease.py:42]: 🙏

**Pull request merged**